src/families/cardano/js-getTransactionStatus.ts

```diff
@@ -0,0 +1,68 @@
+import { BigNumber } from "bignumber.js";
+import {
+  NotEnoughBalance,
+  RecipientRequired,
+  FeeNotLoaded,
+  InvalidAddress,
+  AmountRequired,
+} from "@ledgerhq/errors";
+import type { Account, TransactionStatus } from "../../types";
+import type { Transaction } from "./types";
+import { isValidAddress } from "./logic";
+import { utils as TyphonUtils } from "@stricahq/typhonjs";
+import { getCurrentCardanoPreloadData } from "./preload";
+import { CardanoMinAmountError } from "./errors";
+
+const getTransactionStatus = async (
+  a: Account,
+  t: Transaction
+): Promise<TransactionStatus> => {
+  const errors: Record<string, Error> = {};
+  const warnings: Record<string, Error> = {};
+  const useAllAmount = !!t.useAllAmount;
+
+  const estimatedFees = t.fees || new BigNumber(0);
+  const amount = t.amount;
```

Comment:
Unless I'm mistaken, I don't see where the `useAllAmount` param is used to get the correct amount.
I'm referring to this kind of logic: https://github.com/LedgerHQ/ledger-live-common/blob/5fa6a917da4a4789d757ae67eb0679f54ce6cd52/src/families/bitcoin/js-getTransactionStatus.ts#L96